tests/general/TextLintBearTest.py

```diff
@@ -14,10 +14,12 @@ def get_testfile_path(name):
                         'textlint_test_files',
                         name)
 
+with open(get_testfile_path(name)) as f1:
+    return f1.readlines()
```

Comment:
this should remain in a function, doesn't work otherwise ;)

Reply:
ok,I'll make it thanks,